sdcm/utils/common.py

```
@@ -3043,3 +3043,31 @@ def get_node_disk_usage(node: "BaseNode") -> float:
+        cluster: BaseCluster instance
+
+    Returns:
+        Dictionary with min, max, avg disk usage percentages
+    """
+    import statistics  # noqa: PLC0415 - inline import for convenience
+
+    usages = []
+    for node in cluster.nodes:
+        try:
+            usages.append(get_node_disk_usage(node))
+        except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)